Foundation/IndexSet.swift

```diff
@@ -105,7 +105,7 @@ public struct IndexSet : ReferenceConvertible, Equatable, BidirectionalCollectio
             return IndexingIterator(_elements: self)
         }
         
-        public subscript(index : Index) -> CountableRange<IndexSet.Element> {
+        public subscript(index : Index) -> Range<IndexSet.Element> {
```

Comment:
Do we have this same change in the `IndexSet` in the swift overlay?

Reply:
@parkera - yeah, @airspeedswift took care of the swift overlay

Reply:
Ah right, sorry, I should have set myself a reminder to do it here once we rebranched. Thanks!